core/flyout.js

```diff
@@ -459,10 +612,17 @@ Blockly.Flyout.prototype.layoutBlocks_ = function(blocks, gaps, margin) {
     block.render();
     var root = block.getSvgRoot();
     var blockHW = block.getHeightWidth();
-    var x = this.RTL ? 0 : margin / this.workspace_.scale +
-        Blockly.BlockSvg.TAB_WIDTH;
-    block.moveBy(x, cursorY);
-    cursorY += blockHW.height + gaps[i];
+    var tab = block.outputConnection ? Blockly.BlockSvg.TAB_WIDTH : 0;
+    if (this.horizontalLayout_) {
+      cursorX += tab;
+    }
+    block.moveBy((this.horizontalLayout_ && this.RTL) ? -cursorX : cursorX,
+        cursorY);
+    if (this.horizontalLayout_) {
```

Comment:
Cancel that previous comment.  Git's diff and I are at odds with each other.  Thanks @rachel-fenichel for your patience.